spec/Dynamic-Validator.md

```diff
@@ -43,12 +43,12 @@
 9. No *SELF_NOMINATE* during **RELEASE_PERIOD**
 
 ## Term
-The term is a period when one elected validator set works, and lasts for almost an hour
+The term is a period when one elected validator set works, and lasts for almost an hour.
```

Comment:
A term is a period when one elected validator set works, and lasts for almost an hour.